src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -154,7 +84,11 @@ export const FiltersDrawer = ({
       dismissible={true}
       drawerContainerClassName={containerClassName}
       isActive={isActive}
-      onClose={() => onClose(checkedMap)}
+      onClose={() => {
+        if (onClose) {
+          onClose();
+        }
```

Comment:
I forget – can we do `onClose && onClose();`?

Reply:
Yup, will do that looks cleaner 👍